libr/bin/format/zimg/zimg.h

```diff
@@ -0,0 +1,39 @@
+#include <r_types.h>
+#include <r_util.h>
+#include <r_lib.h>
+#include <r_bin.h>
+
+#define R_BIN_ZIMG_MAXSTR 256
+
+#pragma pack(4)
```

Comment:
what's the purpose of packing this. should you push then pop this pack pragma? like is done here?
http://www.cplusplus.com/forum/general/14659/